app/app.js

```diff
@@ -1,18 +1,16 @@
-import Ember from 'ember';
-import Resolver from './resolver';
+import Ember, { Application } from 'ember';
 import loadInitializers from 'ember-load-initializers';
-import config from './config/environment';
-
-let App;
+import { modulePrefix, podModulePrefix } from 'ember-get-config';
```

Comment:
This is nice for accessing app config from an addon, but seems a bit overkill for something as simple as the dummy app. What is the advantage of relying on it here?